internal/acctest/acctest.go

```diff
@@ -70,7 +70,7 @@ func PreCheck(t *testing.T) {
 		option.WithClientName("terraform-acceptance-tests"),
 		option.WithCSPUrl(cspURL),
 		option.WithAPIKey(apiKey),
-		option.WithDebug(true),
```

Comment:
this should have been true by default for `acctest.go`

Reply:
done